data/remnant missions.txt

```diff
@@ -605,6 +605,7 @@ mission "Remnant: Heavy Laser"
 	on complete
 		outfit "Heavy Laser" -2
 		payment 1000000
+		log "People" "Tali" `As an engineering prefect, Tali is the foremost authority among the Remnant on salvaged technology and a senior leader in their reverse engineering program.`
```

Comment:
Won't this create multiple identical entries? If so then the way to fix this is to create a separate mission that triggers once after any of the appropriate missions giving this entry. (Also, I'm not sure a name reference is a good idea. But whatever.)

Reply:
These missions are mutually exclusive so only one log will ever be written.

Reply:
As Amazinite said, the three missions are mutually exclusive, so that entry will only trigger once.

I'm not sure I understand what you are meaning by the "name reference". Are you expressing doubt about having a log entry for these characters? or something in my implementation of it?

As an FYI, all three of the characters that get logbook entries in this PR are characters that will be used in future missions.

Reply:
I think jafdy was talking about mentioning Tali again in her own entry. But again, I think it's a correct attribution.

Reply:
OK. I'll mark this resolved then.

Reply:
As in referencing a character from another game. Tali, the engineer from the mass effect games. As a rule you need to be very careful if you make such a reference.

Reply:
Yes, that's a given. That being said, the only similarities between the two are a first name and the fact they are engineers. Their situation in life, their personality, description, and everything else is different. (And the more missions with description I make, the more different she's going to be. Although if you'd like me to include a physical description in the log entry to make it clear, I'll be happy to do it. Something to the effect of being tall, ebonny skinned engineer who regularly works on ships in the frost air without benefit of insulated clothing.)

In this regard, I'm taking the Lampyrid as a guide in this. That ship is very distinctly the Serenity, especially with a name that translates as "Firefly". Generally speaking, an image is a lot more distinctive and unique than is a solitary first name. Especially when it is a name with a slightly disguised matching name. Even more so when it comes with a list of quotes that sound very much like quotes from Mal. (I'm rusty on my lore, so I'm not sure if they are actual quotes, but they are very similar)

Reply:
The Lampyrid is an exceptional case, since it's a person ship.

Reply:
If it came down to it, I suspect the fact that one is a "person ship" and the other is an "NPC" would be irrelevant to a copyright lawyer.

I just did some research, and some law firm by the name of [Odin Law](https://odinlaw.com/how-to-add-easter-eggs-to-your-game-legally/) came up (from the description, it is a firm that specializes in software and media.) They have a post on Easter eggs, of which one segment seems relevant:

> Easter eggs done legally
>
>Overall, the concept of Easter eggs are legal. But it can be easy to infringe on copyright or trademarks depending on how an Easter egg is implemented.
>
>If a developer has its own creation and its own design that mimics an existing mechanic or reference in another game, it’s more likely an acceptable reference. If it references the developer’s own prior works, it’s even more likely acceptable.
>
>If a developer extracts an exact model of an asset in another game or clearly recreates its existing design, the developer may be infringing on copyrights of the original creator. Developers cannot and should not copy and paste another’s work. Period.

Going by this description, Tali (my own creation that references another game) should be fine, but the Lampyrid (which clearly recreates an existing design) is problematic. Now, I really don't want to argue for the removal of the Lampyrid, but I am firmly convinced that if the Laympyrid is deemed OK to be included (which it obviously has been), a nod to another of science fiction's famed engineers should be fine.

Reply:
Oh, I wasn't even looking at this from a copyright perspective, I was just thinking about how obvious a reference something is.

Reply:
Oh. So you were referring to the potential problems of having two Tali's who are engineers in two different games? I'm guessing problems of people conflating the two?